### ISSUE TITLE
submitting not defined

### ISSUE THREAD

**Issue by v3yr0n** [reactioncommerce/reaction#2503]
### Expected behavior
Complete credit card payment using payflow

### Actual Behavior
browser console error: submitting not defined

### Steps to Reproduce the Behavior
Do a payment using the payflow method

### Versions
Node: 8.1.2
NPM: 5.0.3
Meteor Node: 4.8.3
Meteor NPM: 4.6.1
Reaction CLI: 0.9.3
Reaction: 1.3.0
Reaction branch: master

Just add `let submitting` at line 11 of the imports/plugins/included/payments-paypal/client/templates/checkout/payflow/payflowForm.js

**Comment by zenweasel** [comment]
I think this may have already been fixed by #2450

**Comment by zenweasel** [comment]
Yeah, that error was fixed but now I am getting another error I am looking into

```
'{"name":"VALIDATION_ERROR","details":[{"field":"transactions[0].amount.total","issue":"Currency amount must be non-negative number and can optionally contain exactly two decimal places separated by \'.\', optional thousands separator \',\', limited to seven digits before the decimal point."}],"message":"Invalid request. See details.","information_link":"https://developer.paypal.com/webapps/developer/docs/api/#VALIDATION_ERROR","debug_id":"d91e6feb9516e"}'
```

**Comment by zenweasel** [comment]
Actually I can't recreate that bug after reset so closing. This issue should be fixed on the `development` branch by #2450

**Comment by v3yr0n** [comment]
ok, thanks!
